system/security/digisign/digisign.cpp

```diff
@@ -304,14 +305,28 @@ extern "C"
 
         if (!isEmptyString(_pubKey))
         {
-            publicKeyBuff.loadFile(_pubKey);
+            try
+            {
+                publicKeyBuff.loadFile(_pubKey);
+            }
+            catch (IException * e)
+            {
+                e->Release();
+            }
```

Comment:
Is this because you don't want -
```
throw MakeStringException(errno, "File %s could not be opened", filename);
```
Which is what is seems is the only throw from loadFile(const char *)

Reply:
Correct, since I was asked not to display the name/location of the key files